### ISSUE TITLE
Icinga2 API call

### ISSUE THREAD

**Issue by rahulramani** [saurabh-hirani/icinga2-api-examples#2]
How to add a parameter to host object

Similar to vars.os i need another vars.disks.disk to be added so my host object should look like

object Host "mons-07.localdomain" {
import "generic-host"
address = "30.41.80.49"
check_command = "hostalive"
         vars.disks["disk"] = {
  }
vars["os"] = "Linux"
version = 1461156108.685005
}

when i add attr in the curl call it generates this host object

object Host "mons-07.localdomain" {
        import "generic-host"

```
    address = "30.41.80.49"
    check_command = "hostalive"
```

*\*        vars["disks"]["disk"] = {
        }**
        vars["os"] = "Linux"
        version = 1461156108.685005
}

attributes that i am passing in curl call
"attrs": { "address": "10.41.80.49", "check_command": "hostalive", "vars.os" : "Linux" , "vars.disks.disk": {} } 

Basically i am getting 
    vars["disks"]["disk"] = {
        }

instead of
         vars.disks["disk"] = {
  }

**Comment by saurabh-hirani** [comment]
I don't think this is the right repo to put in issues because what you've mentioned is not an issue with the examples I've added. This is more of an icinga2 introductory examples repo.  While you've added it, I will check it out, but isn't vars.disks["disk"] the same as vars["disks"]["disk"] in terms of use case? I've not worked with nested attrs so my knowledge might be cursory. But could you explain the diff between the two?

**Comment by saurabh-hirani** [comment]
Create

```
uri: /v1/objects/hosts/mons-07.localdomain 
data: '{ "name": "mons-07.localdomain", "attrs": {"address": "1.2.3.4", "check_command": "hostalive", "vars.disks.disk": {}, "vars.os": "linux" } }'
```

Update

```
uri: /v1/objects/hosts/mons-07.localdomain 
data: '{ "attrs": {"vars.disks.disk": {"disk1": {"a": "b" }} } }
```

Verify

```
uri: /v1/objects/hosts/mons-07.localdomain
```

adds the params for me.
